package.json

```diff
@@ -65,10 +73,12 @@
         "@types/react-router": "^4.4.3",
         "@types/react-router-dom": "^4.3.1",
         "@types/styled-components": "^4.1.6",
+        "coveralls": "^3.0.3",
         "cross-env": "^5.2.0",
         "enzyme": "^3.8.0",
         "enzyme-adapter-react-16": "^1.7.1",
         "husky": "^1.3.1",
+        "jest": "^23.6.0",
```

Comment:
Isn't jest already a dependency through CRA? Probably not a big deal, just curious if this is necessary.

Reply:
I tried to use it before adding this new dependency and did not worked, so I just added it